src/modules/core/components/Fields/Input/Input.tsx

```diff
@@ -135,6 +137,7 @@ const Input = ({
     disabled,
     maxLength,
     maxButtonParams,
+    'data-test': dataTest,
```

Comment:
I did this slightly differently, which is your preference? 

https://github.com/JoinColony/colonyDapp/pull/3241/files#diff-cda2cd04917ac9d136b544add03a27d755e47db79353775e5a0c060e7c6191d4R117

Reply:
Yours look cleaner :+1: I'm gonna check your change out